docroot/modules/contrib/ova/js/helper.js

```diff
@@ -0,0 +1,13 @@
+///**
```

Comment:
We can't use helper.js file. and audio/video.min.js file for audio which we altered for showing poster image over the audio.